clients/drcachesim/tracer/raw2trace.h

```diff
@@ -83,23 +84,11 @@ namespace drmemtrace {
 #    define DEBUG_ASSERT(x) /* nothing */
 #endif
 
-#define OUTFILE_SUFFIX "raw"
 #ifdef BUILD_PT_POST_PROCESSOR
 #    define OUTFILE_SUFFIX_PT "raw.pt"
 #endif
-#ifdef HAS_ZLIB
-#    define OUTFILE_SUFFIX_GZ "raw.gz"
-#    define OUTFILE_SUFFIX_ZLIB "raw.zlib"
-#endif
-#ifdef HAS_SNAPPY
-#    define OUTFILE_SUFFIX_SZ "raw.sz"
-#endif
-#ifdef HAS_LZ4
-#    define OUTFILE_SUFFIX_LZ4 "raw.lz4"
-#endif
-#define OUTFILE_SUBDIR "raw"
+
 #define WINDOW_SUBDIR_PREFIX "window"
```

Comment:
It also seems like this should live with the _FORMAT one: should we move them all as a group.

Reply:
Done.